protocol-designer/src/pages/Designer/utils.ts

```diff
@@ -95,17 +101,25 @@ export const getSlotInformation = (
     .filter(def => def.allowedRoles?.includes('lid'))
     ?.map(def => def.parameters.loadName)
   const offDeckLabware = deckSetupLabware[slot]
+  const isSlotAHopper = getIsSlotAHopper(slot)
+  const adjustedSlot = isSlotAHopper
+    ? FAKE_HOPPER_LOCATION_MAP[slot as HopperLocationMapKey]
+    : slot
   const slotPosition =
     deckDef != null && offDeckLabware == null
-      ? (getPositionFromSlotId(slot, deckDef) ?? null)
+      ? (getPositionFromSlotId(
+          adjustedSlot,
+          deckDef,
+          isSlotAHopper ? HOPPER_LABWARE_X_OFFSET : undefined
+        ) ?? null)
```

Comment:
I don't think we need to nullish coalesce here since `getPositionFromSlotId` can return null right?

```suggestion
      ? getPositionFromSlotId(
          adjustedSlot,
          deckDef,
          isSlotAHopper ? HOPPER_LABWARE_X_OFFSET : undefined
        )
```